spec/feature/unauthenticated_guest_spec.rb

```diff
@@ -213,14 +214,14 @@
     expect(page).to have_content("Page Not Found")
   end
 
-  def click_add_to_cart_link(category)
-    click_link_or_button "Menu"
-    within(".categories") do
-      within("div##{category}") do
-        within("div.item") do
-          click_link "Add to Cart"
-        end
-      end
-    end
-  end
+  # def click_add_to_cart_link(category)
```

Comment:
In the future, you can just add an x next to the "it" for rspec and it will turn it into a pending test